CHANGELOG.rst

```diff
@@ -0,0 +1,34 @@
+Changelog
+========================================================================
+
+in development
+------------------------------------------------------------------------
+
+Added
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* A changelog file.
+
+    Contributed by @nzlosh
+
+Fixed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Fix the CI build caused by different OS version.
+* Fix CircleCI build for forked PRs.
+
+    Contributed by @armab
+
+Changed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Updated hubot-slack to "^4.10.0"
+
+    Contributed by @lopf
+
+Removed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Remove skills-validator from npm-shrinkwrap.json
+
+    Contributed by @armab
+
+
+v0.7.0
```

Comment:
Same as in https://github.com/StackStorm/st2web/pull/989#pullrequestreview-1120786777 problems, I'm thinking how do we tie this information to the actual stackstorm release version.

How about hinting at the respective st2 version in the changelog as well?
```
v0.7.0 (st2chatops v3.7.0)
```

Or maybe better ignore the package.json version and just use the real stackstorm version, because the `v3.7.0` chatops artifact is what really produced based on the https://github.com/StackStorm/st2chatops/ repository build pipeline here.